cef/CustomEditField/CustomEditField.rbbas

```diff
@@ -1190,11 +1190,19 @@ Implements MessageReceiver
 		  if SelLength = 0 then Return
 		  dim c as new Clipboard
 		  
+		  Dim textToCopy as String
+		  
 		  #if EditFieldGlobals.Replace00With01
-		    c.Text = me.SelText.ReplaceAll (Chr(1), Chr(0))
+		    textToCopy = me.SelText.ReplaceAll (Chr(1), Chr(0))
 		  #else
-		    c.Text = me.SelText
+		    textToCopy = me.SelText
+		  #endif
+		  
+		  #if TargetWindows
+		    textToCopy = textToCopy.ReplaceAll(EndOfLine.Macintosh, EndOfLine.Windows)
```

Comment:
If SelText delivers the lines with CR even on Windows, then that's where the bug is, IMO, not in the Clipboard function.
I.e, SelText should always use the native line delimiters, should it not? It should behave the same as a TextField does.